### DISCUSSION TITLE
Search not work

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20804] [General]
It doesn't work on the last one "latest-night-build"!

For example - Serbia, POI "Sijalica Shop"
The POI exists on the map - https://www.openstreetmap.org/node/11745048341

**Comment by DmitryAlexei** [comment]
You need to enable live updates (see https://osmand.net/docs/user/personal/maps-resources#osmand-live)
you can also try online search (https://osmand.net/docs/user/search/search-poi#online-search)

**Comment by ZeljkoBG** [comment]
Thanks for the answer.

**Live udates** - is not needed because the POI has been on the map for a long time.
**Online search** - is not needed because the POI has been on the map for a long time.

I updated today's version and the search works. I found the POI "Sijalica Shop".

I don't know why the search didn't work. But I had to report that the app with the error does not reach the user
